### PR TITLE
Rebase to upstream 12-4-24

### PR DESCRIPTION
**Pull request by michealmouner**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | yes/no
| Deprecations? | yes/no
| Tests pass?   | yes/no
| Fixed tickets | 
| License       | Apache2

## Description
A few sentences describing the overall goals of the pull request's commits.

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Changelog

### CODE REVIEW

**Review by ozahorulia — APPROVED** [review]

**Review by tomekknapczyk — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  phpunit:
```
Comment (balazscsaba2006) [inline .github/workflows/ci.yaml:13]: Why nod add the Coding Standards before the `phpunit` job, in the same workflow?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    strategy:
+      fail-fast: false
+      matrix:
+        symfony-version:
+          - '^4.4'
+          - '^5.4'
+          - '^6.4'
+        php-version:
+          - "7.4"
```
Comment (balazscsaba2006) [inline .github/workflows/ci.yaml:25]: In composer.json you have PHP 7.2, it tests on PHP 7.4, but I would remove support for <8.0, and rewrite the code accordingly (eg. dump `@var` annotations everywhere, etc.); you can use rector for automatically fixing up the codebase.

File: Controller/TranslateController.php [hunk 3]
```diff
@@ -26,7 +26,9 @@
 use JMS\TranslationBundle\Util\FileUtils;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use Twig\Environment;
 
 /**
  * Translate Controller.
```
Comment (balazscsaba2006) [inline Controller/TranslateController.php:34]: Are you actually using the translations admin? I find completely unusable, if no one is using it, this controller with all it's routes should go.

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (balazscsaba2006) [reply .github/workflows/ci.yaml:25]: Also, Symfony version should be the lowest that the monolith is using, `4.4` should not be tested.

**Review by balazscsaba2006 — CHANGES_REQUESTED** [review]

**Review by balazscsaba2006 — APPROVED** [review]

**Pull request merged**